resolwe/test/testcases.py

```diff
@@ -281,7 +281,7 @@ def setUp(self):
         self._register_schemas()
 
         self.collection = Collection.objects.create(contributor=self.admin, name="Test collection")
-        self.files_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'files')
+        self.files_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'toolkit', 'tests', 'files'))
```

Comment:
You can create main test class in toolkit and override it there. It would be probably the best if error is raised if `self.files_path` is not overridden in child class.